core/app/models/spree/ability.rb

```diff
@@ -32,7 +32,7 @@ def initialize(user)
       alias_action :new_action, to: :create
       alias_action :show, to: :read
       alias_action :index, :read, to: :display
-
+      alias_action :create, :update, :destroy, to: :modify
```

Comment:
@JDutil manage implies you can do all actions on an object. For example you may be able to create, edit or destroy an order (therefore view it's edit screen) but you cannot approve an order (which would be included with the manage action). modify implies you can mutates an object BUT not necessarily perform all actions.